tests/playwright/src/model/pages/run-image-page.ts

```diff
@@ -18,8 +18,13 @@
 
 import type { Locator, Page } from '@playwright/test';
 
+import { waitWhile } from '../../utility/wait';
 import { BasePage } from './base-page';
-import { ContainersPage } from './containers-page';
+
+export interface ContainerInteractiveParams {
```

Comment:
This shouldn't be here, it should be moved in one of the other files where interfaces are defined.